test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -10,6 +10,7 @@ def setup
 
     @credit_card = credit_card('4111111111111111', verification_value: '987')
     @declined_card = credit_card('801111111111111')
+    @credit_card_amex = credit_card('6011111111111117', brand: 'discover')
```

Comment:
💬  Comment:
I think this variable is not needed as I see you are not using it in any place.